packages/utils/src/db/db.ts

```diff
@@ -70,8 +71,11 @@ function createPendingAcknowledgement(halfKey: HalfKeyChallenge) {
 function createPacketTagKey(tag: Uint8Array) {
   return Uint8Array.from([...PACKET_TAG_PREFIX, ...tag])
 }
-function createRegistryAccount(address: Address) {
-  return Uint8Array.from([...REGISTRY_ELEGIBLE_ACCOUNT_PREFIX, ...address.serialize()])
+function createWhitelistRegistryKey(publicKey: PublicKey) {
+  return Uint8Array.from([...WHITELIST_REGISTRY_PREFIX, ...publicKey.serializeCompressed()])
```

Comment:
```suggestion
  return Uint8Array.from([...WHITELIST_REGISTRY_PREFIX, ...publicKey.serializeUncompressed()])
```
Please always store publicKeys *uncompressed* because compressing makes EC_SQRT operations necessary because it destroys information

Reply:
fixed